interface/usergroup/addrbook_edit.php

```diff
@@ -87,7 +87,7 @@ function invalue($name) {
   // 2 = Person Centric
   // 3 = Company Centric
   $sql = sqlStatement("SELECT option_id, option_value FROM list_options WHERE " .
-   "list_id = 'abook_type'");
+   "list_id = 'abook_type' AND activity = 1");
```

Comment:
I wish developer would use nouns that actually have some relevance to the use to the value...

Activity = "was there any activity on the thing in the past?"
Active = "Is this thing temporarily available for use or NOT"
Deleted = "Is this thing permanently disabled for users"